var/spack/repos/builtin/packages/py-iniconfig/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyIniconfig(PythonPackage):
+    """
+    iniconfig: brain-dead simple parsing of ini files
+
+    iniconfig is a small and simple INI-file parser module having a unique
+    set of features:
+
+    * tested against Python2.4 across to Python3.2, Jython, PyPy
+    * maintains order of sections and entries
+    * supports multi-line values with or without line-continuations
+    * supports '#' comments everywhere
+    * raises errors with proper line-numbers
+    * no bells and whistles like automatic substitutions
+    * iniconfig raises an Error if two sections have the same name.
```

Comment:
`spack info` removes newline characters, so this likely won't be formatted like you expect. I would just keep the first line description, that's probably enough.